ai_agents/agents/examples/voice-assistant-with-memU/tenapp/ten_packages/extension/main_python/extension.py

```
@@ -268,36 +268,36 @@ async def _retrieve_memory(self, user_id: str = None) -> str:
             )
             return ""
 
-    async def _retrieve_related_memory(self, query: str, user_id: str = None) -> str:
+    async def _retrieve_related_memory(
+        self, query: str, user_id: str = None
```

Ruff (0.14.0)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)